examples/gno.land/p/samourai/daokit/README.MD

```diff
@@ -0,0 +1,290 @@
+# 1. Introduction
+
+A **Decentralized Autonomous Organization (DAO)** is a self-governing entity that operates through smart contracts, enabling transparent decision-making without centralized control.
+
+Daokit is a gnolang package for creating complex DAO models. It introduces a new framework based conditions. BaseDAO is an extension of DAOkit that handles the members & roles through role manager pkg.
+
+# 2. Why daokit ?
+
+The advantages:
+- Role system to represent the organization & responsabilities of the members
+- Condition system to create complex proposals combining roles, counting, percentages, logical operations, etc.
+
+## 2.1 Key Concepts
+
+To understand how it works, here are three fundamental concepts:
+
+- **Role:** A label assigned to a user (or entity) that grants specific **permissions and governance power** based on the organization’s structure.
```

Comment:
I don't think a role is a fundamental concept
proposals are though

Reply:
I think proposal should be added

But i still think role is a fundamental concept of daokit because:
- a Role system/hierarchy is a fundamental concept of any organization (almost)
- Our condition system even not directly linked to roles, get most of this value by the combination with the roles manager

I feel you when you say it's not fundamental as it's not included in the core and daokit can work without roles.
Now role combined with daocond is what makes daokit interesting to external users IMO

I can try to rework it to say we have the core concepts (resources, proposals, conditions) that is required to have a working DAO and say that we plug a role manager pkg to exploit the maximum of the core features.